language/tools/test-generation/src/lib.rs

```diff
@@ -44,22 +45,12 @@ fn run_verifier(module: CompiledModule) -> Result<VerifiedModule, String> {
 
 /// This function runs a verified module in the VM runtime
 fn run_vm(module: VerifiedModule) -> Result<(), String> {
-    let modules = ::stdlib::stdlib_modules().to_vec();
```

Comment:
why did you decide to go down this path? that is, change the code to `execute_function_in_module`?
Just curious and fine this way

Reply:
I think it makes more sense to use the same code path as the rest of the functional_tests/e2e_test, during which we start the execution from the genesis state. And I think the `execute_function_in_module` is a more abstract api we can use for all the tools that we have. The idea is that we can start execute a function in a module at a given blockchain state, which is a pattern used by both the test_generation and cost_synthesis.